src/pages/SignUpVolunteer.js

```diff
@@ -31,17 +31,21 @@ function SignUpResearcher() {
   const classes = useStyles();
   const [sent, setSent] = React.useState(false);
 
-  const validate = (values) => {
+  const validate = values => {
     const errors = required(
-      ["organization", "faculty", "shortIntroduction","age","nationality"],
+      ["organization", "faculty", "shortIntroduction", "age", "nationality"],
       values
-    )
+    );
 
     return errors;
   };
 
-  const handleSubmit = (values) => {
+  const handleSubmit = values => {
     //TODO: authenticate to allow login
+
+    // Db.addVolunteerData(useContext(UserContext)?.uid, ...); TODO
+    // navigate to next page
```

Comment:
nit: prefers to not leave commented codes in production code. just for future reference :)